index.html

```diff
@@ -1557,6 +1557,16 @@ <h3>Issuance Date</h3>
   "proof": { ... }
 }
         </pre>
+
+        <p class="note">
+It is expected that the next version of this specification will rename the
+<code>issuanceDate</code> <a>property</a> to <code>validFrom</code> in a way
+that is backwards-compatible. The range of acceptable values will remain the
+same as will the semantic meaning of the <a>property</a>. Implementers are
+advised that the <code>validFrom</code> <a>property</a> is reserved and its use
+for any other purpose is discouraged.
```

Comment:
I think there's a slight problem here for JWT representation. We current express `issuanceDate` as the ["iat" claim](https://tools.ietf.org/html/rfc7519#section-4.1.5) in a JWT. The `validFrom` property should instead map to the ["nbf" claim](https://tools.ietf.org/html/rfc7519#section-4.1.6) in a JWT.

It is true that we intended "issuanceDate" to *mean* `validFrom` in the absence of that property. However, I think *both* properties could be present in a VC with different values and, under that circumstance, `validFrom` should take precedence for determining validity. So `issuanceDate` means the earliest point at which a VC should be considered valid, unless `validFrom` is present. Of course, we can't say that normatively now and we're just reserving this for the future.

So I think it's still true that the range of acceptable values will remain the same and the semantic meaning of the property will be the same but this will not be a "renaming", rather, the presence of `validFrom` would take precedence over `issuanceDate` when determining the VC validity period. Also, as mentioned above, mapping these properties to a JWT would result in using different JWT claims.

cc: @TallTed

Reply:
Good catch.  I think that in the current iteration, we must express `issuanceDate` as `nbf` (as this is and was the intended semantic), and not as `iat` (which was never the intended semantic).  Future versions of the VCDM can change things however is appropriate at that point, including how precedence lays out if/when both are present.

Reply:
Ok, updating the spec w/ these changes in mind.

Reply:
Fixed in afaedca614da265795c894714bee97b805dd48e6